COMET.Web.Common/Model/Configuration/ServerConfiguration.cs

```diff
@@ -0,0 +1,36 @@
+// --------------------------------------------------------------------------------------------------------------------
+//  <copyright file="ServerConfiguration.cs" company="RHEA System S.A.">
+//     Copyright (c) 2023 RHEA System S.A.
+// 
+//     Authors: Sam Gerené, Alex Vorobiev, Alexander van Delft, Jaime Bernar, Théate Antoine
+// 
+//     This file is part of COMET WEB Community Edition
+//     The COMET WEB Community Edition is the RHEA Web Application implementation of ECSS-E-TM-10-25 Annex A and Annex C.
+// 
+//     The COMET WEB Community Edition is free software; you can redistribute it and/or
+//     modify it under the terms of the GNU Affero General Public
+//     License as published by the Free Software Foundation; either
+//     version 3 of the License, or (at your option) any later version.
+// 
+//     The COMET WEB Community Edition is distributed in the hope that it will be useful,
+//     but WITHOUT ANY WARRANTY; without even the implied warranty of
+//     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+//    Affero General Public License for more details.
+// 
+//    You should have received a copy of the GNU Affero General Public License
+//    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//  </copyright>
+//  --------------------------------------------------------------------------------------------------------------------
+
+namespace COMET.Web.Common.Model.Configuration;
+
+using System.Text.Json.Serialization;
+
+public class ServerConfiguration
```

Comment:
Is this class used? If the intent is to have a class that have all configuration (That I fully agree on), please use it and directly deserialize the content on the service. You can also refactor a bit the ConfigurationService to only have this instance of the object. Will be easier to maintain later.

Reply:
It's used on the ConfigurationService implementation for the Server side. I can modify the WASM implementation so that it utilizes this aswell, instead of separate properties. What do you think?

Reply:
Please remove properties on the IConfigurationService and replace them by an instance of ServerConfiguration